keyboards/binepad/bn006/info.json

```
@@ -0,0 +1,40 @@
+{
+    "manufacturer": "Binepad",
+    "keyboard_name": "BN006",
+    "maintainer": "Binepad",
+    "bootloader": "stm32duino",
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic_lite": true,
```

There is no bootmagic lite, anymore. Just bootmagic. Which uses the lite code. 
```suggestion
        "bootmagic": true,
```